nomos-services/api/src/lib.rs

```diff
@@ -10,6 +12,8 @@ use overwatch::{
 
 pub mod http;
 
+pub(crate) const HTTP_REQUEST_TIMEOUT: Duration = Duration::from_secs(10);
```

Comment:
I think this would be better to load from configuration, at the moment we don't know how much some longrunning operations could take, it will be a bit easier to test it with different timeout settings.

Reply:
Should be fixed now. I am not sure if there was a better way, but I tried to keep everything in the ApiService.